packages/flutter_tools/lib/src/base/command_help.dart

```diff
@@ -35,6 +36,13 @@ class CommandHelp {
 
   final OutputPreferences _outputPreferences;
 
+  CommandHelpOption _exclamationPoint;
+  CommandHelpOption get exclamationPoint => _exclamationPoint ??= _makeOption(
+    '!',
```

Comment:
'I' also looks available. Big 'I' for a big image warning makes some sense, I think.

Reply:
It currently does the widget inspector.  If we're ok with changing that I'll use it.

Reply:
Lets not change the existing bindings if possible

Reply:
Ah, okay. The help message only claims 'i' for the widget inspector, but the resident runner triggers it with both 'I' and 'i'. I would be okay with keeping 'i' for the widget inspector and using 'I' for this. @jonahwilliams thoughts?

Reply:
That is, I would be okay with the help message being the source of truth about what they keys do, rather than what the resident runner implements.

Reply:
SGTM, if its not documented it might have been an oversight/mistake at some point.

Reply:
Ok, I'll take over upper case I for this.